### ISSUE TITLE
Double-click to open image from queue

### ISSUE THREAD

**Issue by Beep6581** [Beep6581/RawTherapee#1306]
Originally reported on Google Code with ID 1322
```
Photos already added to the queue can not be opened from the queue. It would be great
if double click could open the already queued images.
```

Reported by `gyurko.david@e-arc.hu` on 2012-04-12 21:57:12

**Comment by adamreichold** [comment]
I think this can be closed as it seems like it is already implemented, at least as of version `4.2.476`.

**Comment by Beep6581** [comment]
Confirmed
